.ci/Jenkinsfile

```diff
@@ -67,6 +68,16 @@ pipeline {
         }
       }
     }
+    stage('Release') {
+      when {
+        buildingTag()
```

Comment:
What's the tag strategy? In this case, it will happen for each tag, so in case you'd like to distinguish with some regex/pattern

Reply:
There is semver limitation: https://goreleaser.com/limitations/semver/

Reply:
I'd say to apply the same pattern in the when condition, something like:

```suggestion
        tag pattern: '(v)?\\d+\\.\\d+\\.\\d+', comparator: 'REGEXP'
```

v1.2.3 or 1.2.3 are supported

Reply:
Thanks for suggestion!